tests/cassettes/legendastv/test_list_subtitles_episode_alternative_series.yaml

```diff
@@ -0,0 +1,5985 @@
+        29m/sZU/b4toOnwzGuvizerSj05P5jZhlzlXg6OBkJuiQ5tkMseI/NX8acRZTYou02hORbKKGJz3
+        s1WW8sA3amv47z6Dk8n631DUxRUa0F/4ljLfnLVRxZOTqm9Txe5vfbpKqtca4UtrLW7wly5188mq
+        x10VNV0ScrLbLRpi5WpM4oO+RcLCOeCYTuam5I71cZkvzRBpC1trjVVUK8n4s8zW5kS1MUs/stBT
+        5g76D67FqwXTa72k+csGGEMQYyyWtX5kdWhR9bUu+lWstMHNxviplmk4laUEjmBmQBY1J9zhMLL0
+        9ppx+Z0nE40qommW7X25Ta73LfryONzZ0gSrlihz9wXzv8TbV7VE92XpzXTqlnyKZocQsvOxaDrf
+        7zHF6enwhvKaSzgFPV400S6xDPXKxbzuyLW+PzlRxQlYnzQfI9KIqqDl1yrAszrT60xkjbtNtORW
+        EofDm702aEO4jPfq6o073xgsR3utnYC8pTXVQzgs4vlAqpFsqn0BuN/utcNntt0Oue12xI0F+w92
+        URk9HCV9ejpfu9N/UZur3XpsMryMrLlf+u3ud+r6t5U3y2m7nC8beDs0Nq++ElIsjwPuMcVhBJdz
+        4oirBfHX/S/yrY/J/XzWlfeCZPLazsqMPScwlWZrjZEY1abS3cQ3xbYIm5J9h04eO1PgN42IbUWl
+        l3yzKEUAgHD5JHacoVLf2StT/QdEF5f33rKXLy9OL07OroZP7tfb7fWlUcbLFXhE0WNJfjW2xiTa
+        3W7vO83Vou8WeFSVOlX7p+EkM1++6zSoKfKVYtDeJT6jn6S1OaEuz8vEN6vOZhx7O47vxEVgiOAn
+        6AiWLnVO6vO8K7pQohnasoIT6nFVaDVQ5xZeOro570PN3SR9263pxNJFmLx4NwUCd0vLfRlPRF5f
+        1WsAwnmUpvRsPieVa78Q1ESLeOOL3OHVq8uaaOgEnoSl098kvu4c2jhRawWkwkn0Ku/AhxQOpYAt
+        IPKASO1yNsuYAZ/FbTBzPeZSd2m3P9Zjb3xo72lpqI9o6RkrLdlHVal9AyUxsLx8NgV3Vaat63Kh
+        L9mkTL0lE0nvfx8N0bvr0b/XI2Q8npC2PCxvPv9E5JNYR0JC8fmncRAsbrqd+asNjTkXKlZtydxv
+        yb2UZzvoPLMpL7cfg/EShIOo99DX+HCPnJNgePpAmm7946dV0bR6/Re1X3rNRxfR+KG1RiGdbtjd
+        ndPw/SbhH0xJ+vMPN4df7QsRWvx5p0v+cwN6fF0nSZM4z4TDMSBUX9Tw+mc7s+qMr0Nopm9MPrfq
+        VKf4P3P+21sqK91zWyv1JiJTmTSCyH1bKngrt4iDNNMlIUOLkvx+FarUpnP/Va7lXGuttZ/q/b7/
+        Tl7EQ5ntZ+ZaStDUlZ3N9y++pB3HYYjnddjvHzQDWP5B+8GO1iQcZycoM5M2+tsZaH3E0xrt971J
+        /VIiREzgsRFxP7RWVxK02u32ThHa1mRKTtsZCYwG6p3RU/U+3awWf0daSZ2X2WJ+t9b7M52W3zg9
+        xIhjQP5g+ygTdc/10OwVL+7EVvSZwKNfK49c9f3mn2BXIqtmWGa33HQuLw7Ri1PS26Xoe8GdRjOU
+        QKNHA8xS326fR0z5O2e4F7QllSk8Oedk2iq3/kCjIackIU0XXDiiznAEjd/8F9MsuNtoZhInRva9
+        Gju3PhUkEdXThaUYbJWKy55BBh56LDHmuHVyra/rrohj1cUfhW5IRRDHP0PFsmpPEql3wcSTCmB4
+        12yBiQeIvIr2xYooaDWfmT3DUAEbtT9ofS9GCD/9n0Tv+72g832U+K//AwUql5ZogQAA
+    headers:
+      Accept-Ranges: [bytes]
+      Access-Control-Allow-Origin: ['*']
+      Age: ['0']
+      Connection: [keep-alive]
+      Content-Encoding: [gzip]
+      Content-Length: ['9399']
+      Content-Type: [text/html; charset=utf-8]
+      Date: ['Thu, 29 Jun 2017 20:19:55 GMT']
+      Vary: ['User-Agent,Accept-Encoding']
+      Via: [1.1 varnish]
+      X-Cache: [MISS]
+      X-Cacheable: ['YES:Forced']
+      X-Varnish: ['1389391337']
+      X-backend: [default_director]
+    status: {code: 200, message: OK}
+- request:
+    body: null
+    headers:
+      Accept: ['*/*']
+      Accept-Encoding: ['gzip, deflate']
+      Connection: [keep-alive]
+      Cookie: [PHPSESSID=8ae41jk8qo4omid6gkmduu9dg3; au=3673290---198d5f2e1a202d0c2878ae871f911763d7cd76c0]
+      User-Agent: [Subliminal/2.1]
+    method: GET
+    uri: http://legendas.tv/util/carrega_legendas_busca_filme/44435/1/-/1
```

Comment:
@ratoaq2 can you rebase and use this URI please: http://legendas.tv/legenda/busca/-/1/-/0/44435
thanks